src/queues/job-queue.js

```diff
@@ -1,3 +1,6 @@
+const Sentry = require('@sentry/node');
+const SentryProfiling = require('@sentry/profiling-node');
+const {initializeApp, cert} = require('firebase-admin/app');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of Sentry and SentryProfiling modules has been removed. If these modules are not used in this file, then it's a good change. However, if they are used somewhere else in the code, this will cause a runtime error.
```diff
- const Sentry = require('@sentry/node');
- const SentryProfiling = require('@sentry/profiling-node');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->